restapi/src/test/java/io/stargate/web/resources/WhereParserTest.java

```diff
@@ -0,0 +1,267 @@
+/*
```

Comment:
why does it show this file as changed?

Reply:
I moved it and it got reformatted. It was previously "invisible" to the build.